### PR TITLE
Update node-update.sh

### PR DESCRIPTION
**Pull request by Grothesk242**
Added nodejs20 as the default version

### CODE REVIEW

File: node-update.sh [hunk 1]
```diff
@@ -3,22 +3,23 @@
...
 SYSTDDVIRT=$(systemd-detect-virt 2>/dev/null);
 DOCKER=/opt/scripts/.docker_config/.thisisdocker #used to identify docker
 
 if [ -f "$DOCKER" ];
 then
-        echo "Updating Node.js in Docker is not supported, please update your Docker Container";
+        echo "Fixing Docker is not supported, please update your Docker Container";
+        unset LC_ALL;
+        exit 1;
+        elif [ "$(id -u)" -eq 0 ];
```
Comment (foxriver76) [inline node-update.sh:18]: are these indentations are on purpose?

File: node-update.sh [hunk 2]
```diff
@@ -28,12 +29,18 @@ if [[ $SYSTDDVIRT = "wsl" ]];
...
+        unset LC_ALL;
+        exit 1;
+fi;
 
 ### Starting the skript
 echo -e "ioBroker node-update v$VERSION is starting. Please be patient!";
 HOST=$(hostname)
-NODERECOM=$(iobroker state getValue system.host."$HOST".versions.nodeNewestNext);  #reading node version from iob states. If successful, no fallback required.
-if [[ $NODERECOM != [[:digit:]]*.[[:digit:]]*.[[:digit:]]* ]]; #check if a semver'd nodejs installation is found
+NODERECOM=$(iobroker state getValue system.host."$HOST".versions.nodeNewestNext);  #reading node version from iob states. If succesfull, no fallback required.
```
Comment (foxriver76) [inline node-update.sh:42]: The previous version of `successful` was actually the correct one afaik

File: node-update.sh [hunk 3]
```diff
@@ -231,7 +252,7 @@ if
                                 echo -e "*** Deleting $PATHCOREPACK ***";
                                 $SUDOX rm "$(type -p corepack)";
                 fi
-                echo -e "\nWrong paths have been fixed. Run 'iob diag' or 'iob nodejs-update' again to check if your installation is fine now";
+                echo -e "\nWrong paths have been fixed. Run 'iob diag' or 'iob node-fix' again to check if your installation is fine now";
```
Comment (foxriver76) [inline node-update.sh:255]: is this a existing command `node-fix`?

File: node-update.sh (hunk 1 above)
Comment (Grothesk242) [reply node-update.sh:18]: Do they hurt? But I can try a different layout.

File: node-update.sh (hunk 3 above)
Comment (Grothesk242) [reply node-update.sh:255]: Corrected to node-update

File: node-update.sh (hunk 2 above)
Comment (Grothesk242) [reply node-update.sh:42]: Corrected.

**Comment by Grothesk242** [comment]
Sorry, ich kapier nicht wie man PRs editiert. Hab 'nen neuen gestellt.

**Pull request closed without merging**